packages/synapse-react-client/.storybook/preview.tsx

```diff
@@ -51,57 +51,6 @@ if (process.env.NODE_ENV === 'development') {
   })
 }
 
-export const parameters = {
```

Comment:
Moved these inside of a typed `Preview` object, which provides better typing for these Storybook-defined properties